File: fizzbuzz.rs
/* Rust */
fn main()  {
  for i in 1..=100 {
     let mut line = String::from("");
     if i % 3 == 0 {
       line.push_str("Fizz");
     }
     if i % 5 == 0 {
        line.push_str("Buzz");
     }
     if line.is_empty() {
        println!("{}",i);
     } else {
        println!("{}",line);
     }
  }
}

File: fizzbuzz.ts
let i: number;
for (i = 1; i <= 100; ++i) {
    let line = '';
    if (i % 3 == 0) {
        line += 'Fizz';
    }
    if (i % 5 == 0) {
        line += 'Buzz';
    }
    if (!line) {
        line = '' + i;
    }
    console.log(line);
}
